src/cmd/purge/scripts/onedrivePurge.ps1

```diff
@@ -167,7 +167,8 @@ function Delete-LibraryByPrefix {
 
         if ($PurgeBeforeTimestamp -gt $createTime) {
             foreach ($p in $FolderPrefixPurgeList) {
-                if ($listName -like "$p*") {
+                # also delete list containing
+                if ($listName -like "$p*" -or $listName -like "*$p*") {
```

Comment:
why the full text match?  Shouldn't we only be looking for prefixes?

Reply:
while pre-cleanup/purge, wanted to remove lists from previous run with prefix `Corso_Test_Sanity_` as wells as remove restored lists with prefix `Corso_Restore_Corso_Test_Sanity_`.

Reply:
How about we change `Corso_Restore_Corso_Test_Sanity_` to `Corso_Test_Sanity_Restore_Corso_Test_Sanity_`? Ideal would be `Corso_Test_Sanity_Restore_`, but I'm guessing that will be hard.